<a href="https://colab.research.google.com/github/Monisha-A-M/ASSAIGNMENT-1/blob/main/case_study1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [3]:
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns


# ***TO READ DATASET TO PYTHON***

In [4]:
data=pd.read_csv('cars_data.csv')
data.head()

Make    Model    Nickname  ... 5-yr Depreciation Top Speed  0-60 Time
0  Suzuki   Vitara  Philippine  ...              0.13     200.9       11.5
1   Honda    S2000      Henrik  ...              0.02     158.5        8.8
2     BMW       Z4      Lebbie  ...              0.24     149.5       14.4
3  Toyota   Tacoma    Jefferey  ...              0.20     153.3       11.5
4    Ford  Festiva     Glendon  ...              0.18     122.0       14.9

[5 rows x 20 columns]

we use head() to display only four rows of the given table .
inorder to display the complete table we called the variable name data.

In [ ]:
data

Make        Model    Nickname  ... 5-yr Depreciation Top Speed  0-60 Time
0      Suzuki       Vitara  Philippine  ...              0.13     200.9       11.5
1       Honda        S2000      Henrik  ...              0.02     158.5        8.8
2         BMW           Z4      Lebbie  ...              0.24     149.5       14.4
3      Toyota       Tacoma    Jefferey  ...              0.20     153.3       11.5
4        Ford      Festiva     Glendon  ...              0.18     122.0       14.9
...       ...          ...         ...  ...               ...       ...        ...
9995    Mazda      Tribute      Cooper  ...              0.14     245.8        8.0
9996      GMC  Sierra 2500         Dud  ...              0.11     147.4       12.3
9997  Mercury      Mariner     Chariot  ...              0.12     198.1       11.8
9998   Daewoo      Leganza       Dynah  ...              0.04     206.7       12.2
9999   Toyota      Sequoia    Zedekiah  ...              0.07     242.1        3.2

[10000 rows x 20 columns]

# ***CHECKING FOR NULL VALUE IN DATASET***

In [ ]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 10000 entries, 0 to 9999
Data columns (total 20 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   Make               10000 non-null  object 
 1   Model              10000 non-null  object 
 2   Nickname           10000 non-null  object 
 3   Car Gender         10000 non-null  object 
 4   Buyer Gender       10000 non-null  object 
 5   Buyer Age          10000 non-null  int64  
 6   Buzzword           10000 non-null  object 
 7   Country            10000 non-null  object 
 8   City               10000 non-null  object 
 9   Dealer Latitude    10000 non-null  float64
 10  Dealer Longitude   10000 non-null  float64
 11  Color              10000 non-null  object 
 12  New Car            10000 non-null  bool   
 13  Purchase Date      10000 non-null  object 
 14  Sale Price         10000 non-null  float64
 15  Discount           10000 non-null  float64
 16  Resell Price       1000

since the non-null values are all non zero and equal under each category we can conclude that there are no null entries in the given data

# ***TO PLOT GRAPH BETWEEN FEMALE AND MALE BUYERS***

In [8]:
new_data=data.loc[:,['Make','Buyer Gender']]
new_data

Make Buyer Gender
0      Suzuki       Female
1       Honda         Male
2         BMW       Female
3      Toyota         Male
4        Ford         Male
...       ...          ...
9995    Mazda         Male
9996      GMC         Male
9997  Mercury         Male
9998   Daewoo       Female
9999   Toyota         Male

[10000 rows x 2 columns]

In [ ]:
female=new_data['Buyer Gender']=='Female'
male=new_data['Buyer Gender']=='Male'


# ***TO FIND TOP CARS BASED ON SELL PRICE***

In [20]:
data['Sale Price'].value_counts()

97574.74    2
64443.76    2
23810.58    2
68511.89    2
80084.82    2
           ..
98355.95    1
24866.23    1
36549.25    1
99764.34    1
58859.56    1
Name: Sale Price, Length: 9994, dtype: int64

the highest sell price is 99764.34

# ***TO FIND LEAST SOLD CAR BASED ON RESELL PRICE***

In [21]:
data['Resell Price'].value_counts()

10159.85    2
10046.76    2
26912.51    2
8824.84     2
8914.40     2
           ..
41144.69    1
36882.43    1
11443.77    1
37203.04    1
9429.78     1
Name: Resell Price, Length: 9988, dtype: int64

the lowest resell price is 10159.85